src/components/TimeStampingCell/index.tsx

```diff
@@ -27,10 +30,18 @@ const TimeStampingCell = ({ event }: TimeStampingProps) => {
     }
   }, [setCivility, setLastName, setEndDate, setStartDate, event]);
 
-  const goToManualTimeStamping = (eventStart: boolean) => navigation.navigate(
-    'ManualTimeStamping',
-    { event: { _id: event._id, customer: { identity: event.customer.identity } }, eventStart }
-  );
+  const goToManualTimeStamping = (eventStart: boolean) => {
+    navigation.navigate(
+      'ManualTimeStamping',
+      { event: { _id: event._id, customer: { identity: event.customer.identity } }, eventStart }
+    );
+  };
+
+  useEffect(() => {
+    setStartTimeStamped(
+      event.histories.some((history: any) => history.action === 'manual_time_stamping' && !!history.update.startHour)
```

Comment:
cf mon commentarie sur la PR precedente, que penses tu de réelement typer le history ici -> peut etre definir un EventHistoryType dans le fichier EventType ?

Reply:
Oui j'y ai pense lorsque tu as fait la remarque dans l'autre PR. Je vais créer un nouveau type

Reply:
J'ai un soucis avec le tapage de EventHistory, au moment d'appeler la méthode  `some` dans  event?.histories?.some , j'ai l'erreur suivante: `Property 'some' does not exist on type 'EventHistoryType'.`. Comment pourrais-je l'écrire pour pourvoir utiliser cette methode?

Reply:
J'ai aussi une erreur sur la facon dont j'ai définit le type de StartDate: `Property 'startHour' does not exist on type '{ startHour: Date; }[]'. `